mkdocs/docs/encodings.md

```diff
@@ -2,153 +2,138 @@
+Shared dictionary encoding allows multiple columns to share a common dictionary. Nested fields can also use a shared dictionary (e.g., for localized values like `name:*` columns in an OSM dataset).
+If used for nested fields, all fields sharing the dictionary must be grouped together in the file and prefixed with the dictionary.
 
-Shared dictionary-encoded nullable string columns consists of the following streams and order:
+A shared dictionary-encoded nullable string column consists of the following streams in order:
 `Length`, `Dictionary`, `Present1`, `Data1`, `Present2`, `Data2`
 
-Shared FSST Dictionary-encoded nullable string columns consists of the following streams and order:
+A shared FSST dictionary-encoded nullable string column consists of the following streams in order:
 `SymbolLength`, `SymbolTable`, `String Length`, `Dictionary` (Compressed Corpus), `Present1`, `Data1`, `Present2`, `Data2`
 
 ### Vertex Dictionary Encoding
 
-Uses an additional `VertexOffsets` stream for storing the indices to the coordinates of the vertices
-in the `VertexBuffer` stream. The vertices in the VertexBuffer are sorted on a Hilbert-Curve
-and delta-encoded in combination with a null suppression technique.
+Uses an additional `VertexOffsets` stream to store indices for vertex coordinates in the `VertexBuffer` stream.
+Vertices in the VertexBuffer are sorted using a Hilbert curve and delta-encoded with null suppression.
 
 #### Morton Vertex Dictionary Encoding
 
-The coordinates of the `VertexBuffer` are transformed to a 1D integer using the Morton Code.
-Then the data are sorted according to the Morton code and further compressed by applying
-an Integer compression technique.
+`VertexBuffer` coordinates are transformed into a 1D integer using Morton code.
+The data is sorted by Morton code and further compressed using integer compression techniques.
 
 ## Integer Encodings
 
-In general most data in MLT are stored in the form of arrays of integers.
-Using efficient and fast algorithms for compressing arrays of integers is therefore crucial.
+Most data in MLT is stored as integer arrays, making efficient integer compression crucial.
 
-### Logical Level Technique
+### Logical-Level Techniques
 
-Integers are encoded based on the two lightweight compression techniques delta and rle as well
-as a combination of both schemes called delta-rle.
+Integers are encoded using delta encoding, run-length encoding (RLE), or a combination of both (delta-RLE).
 
-#### Delta
+#### Delta Encoding
 
-Compute the differences between consecutive elements of a sequence e.g. x2 - x1, x3 - x2, ...
-Is always used in combination with a physical level technique to reduce the number of bits needed to store the
-delta values.
+Computes differences between consecutive elements (e.g., x₂ - x₁, x₃ - x₂, ...).
+Used with physical-level techniques to reduce the number of bits required for storing delta values.
```

Comment:
It is not clear that we refer to the techniques below.
When I was originally reading this, I thought this was a convoluted way of saying: "Values are bitpacked".

Linking those sections would likely help ^^

Reply:
not sure what you mean. Delta encoding is fairly well defined in the industry, and at one point I even made this https://crates.io/crates/delta-encoding - what kind of a change you think would help?

Reply:
done